server/backends/migration_cache/migration_cache.go

```diff
@@ -21,8 +24,11 @@ var (
 )
 
 type MigrationCache struct {
-	Src  interfaces.Cache
-	Dest interfaces.Cache
+	Src                  interfaces.Cache
+	Dest                 interfaces.Cache
+	DoubleReadPercentage float64
+
+	mu sync.RWMutex
```

Comment:
Sounds good I'll remove. Was trying to prevent a race condition if there's situations when src and dest cache are temporarily out of sync during a double write. But it's rare that a given key is overwritten right? If data in the CAS is updated, it would be keyed under a new digest?